src/Microsoft.VisualStudio.Threading.Analyzers.CodeFixes/FixUtils.cs

```diff
@@ -298,7 +300,7 @@ internal static AnonymousFunctionExpressionSyntax MakeMethodAsync(this Anonymous
         internal static async Task<Tuple<SyntaxAnnotation, Document, T, SyntaxNode>> BookmarkSyntaxAsync<T>(Document document, T syntaxNode, CancellationToken cancellationToken)
             where T : SyntaxNode
         {
-            var bookmark = new SyntaxAnnotation();
+            var bookmark = new SyntaxAnnotation(BookmarkAnnotationName);
```

Comment:
Can you tell me the significance of this change?

Reply:
I needed to be able to remove the annotation by name to avoid duplicate annotations in the tree.